### PR TITLE
Always run a mandatory cgroup v2 storage job

### PR DESCRIPTION
**Pull request by vasiliy-ul**
Since https://github.com/kubevirt/kubevirt/pull/6042 is merged into the main branch, it probably makes sense to enable at least one presubmit storage job for testing the functionality (i.e. hotplug disks on cgroup v2 host).

### CODE REVIEW

**Comment by vasiliy-ul** [comment]
@iholder-redhat , @rmohr , @fgimenez WDYT?

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -414,13 +413,9 @@ presubmits:
...
         env:
         - name: TARGET
-          value: k8s-1.20
+          value: k8s-1.20-sig-storage
         - name: KUBEVIRT_CGROUPV2
           value: "true"
-        - name: KUBEVIRT_E2E_SKIP
-          value: Multus|SRIOV|GPU|Macvtap|MediatedDevices
-        - name: KUBEVIRT_STORAGE
-          value: rook-ceph-default
```
Comment (iholder-redhat) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:423]: Why is this removed? Most of the hotplug tests that are relevant to cgroups v2 need rook-ceph in the cluster (they are skipped otherwise)

**Review by iholder-redhat — COMMENTED** [review]
I'm very much for it :+1: 
Also wanted to do it but you're faster than me :)

Thanks!

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml (hunk 1 above)
Comment (vasiliy-ul) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:423]: `KUBEVIRT_STORAGE` var is set accordingly  in https://github.com/kubevirt/kubevirt/blob/314b515473a3720dce35f5477c6dbfdc7d544e5c/automation/test.sh#L52

when the `TARGET` is `k8s-*-sig-storage`

Comment (iholder-redhat) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:423]: Thanks!

**Comment by iholder-redhat** [comment]
/lgtm
Thank you

@dhiller FYI

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 2]
```diff
@@ -385,7 +385,7 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
-  - always_run: false
+  - always_run: true
```
Comment (rmohr) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:388]: Can we make it always run but optional for now? Just for this week to see how it behaves?

**Review by rmohr — COMMENTED** [review]
In general great.

@dhiller @brianmcarey can we enable it or do we first have to add more machines? I understood that we are hitting some CI limits in general at the moment.

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml (hunk 2 above)
Comment (vasiliy-ul) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:388]: sure. restored `optional: true`

**Comment by rmohr** [comment]
/lgtm
/approve

/hold

Just giving @dhiller @brianmcarey  time to object if needed.

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 3]
```diff
@@ -401,7 +401,7 @@ presubmits:
       preset-docker-mirror-proxy: "true"
       preset-shared-images: "true"
     max_concurrency: 11
-    name: pull-kubevirt-e2e-k8s-1.20-cgroupsv2-rook-ceph
+    name: pull-kubevirt-e2e-k8s-1.20-sig-storage-cgroupsv2
```
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:404]: Is there a reason why we are using 1.20 here?

Comment (vasiliy-ul) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:404]: No specific reason. It was initially configured for 1.20. Do you think we may need to switch to some other version?

Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:404]: As 1.20 will reach EOL soon, I think we should bump to latest. Are there any objections, @iholder-redhat @vasiliy-ul ?

Comment (iholder-redhat) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:404]: Nope, seems fine by me :+1:

**Comment by iholder-redhat** [comment]
/lgtm
Thank you

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml (hunk 3 above)
Comment (vasiliy-ul) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml]: No objections from my side. Switched to `1.23`

Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml]: Thanks! Taking this in now.
